.github/workflows/git-commit-checks.py

```diff
@@ -195,9 +196,11 @@ def _is_entirely_submodule_updates(repo, commit):
             # These errors mean that the git library recognized the
             # hash as a valid commit, but the GitHub Action didn't
             # fetch the entire repo, so we don't have all the meta
-            # data about this commit.  Bottom line: it's a good hash.
-            # So -- no error here.
-            pass
+            # data about this commit. This occurs because the commit
+            # only exists in a pull request on github. Therefore, we
+            # want to fail this commit until the referenced commit
+            # is merged.
```

Comment:
Technically, commits are not merged: pull requests are merged.  How about:

```suggestion
            # data about this commit. This occurs because the commit
            # only exists in an as-yet unmerged pull request on github. Therefore, we
            # want to fail this commit until the corresponding pull request
            # is merged.
```

(feel free to re-flow the paragraph to make the line breaks appropriate)